mne/label.py

```diff
@@ -2020,6 +2036,147 @@ def read_labels_from_annot(subject, parc='aparc', hemi='both',
     return labels
 
 
+def _check_labels_subject(labels, subject, name):
+    _validate_type(labels, (list, tuple), 'labels')
+    for label in labels:
+        _validate_type(label, Label, 'each entry in labels')
+        if subject is None:
+            subject = label.subject
+        if subject is not None:
```

Comment:
Ah, `label.subject` can be `None`? Maybe comment that so nobody tries to make an `else` out of this

Reply:
There is a referral to this possibility four lines away (in the error message), and a few other places in the code that `if label.subject is (not) None` shows up so it should be safe enough I think.

Reply:
... added a comment anyway